.github/workflows/check-query-files-and-compilation.yml

```diff
@@ -47,12 +47,12 @@ jobs:
       - name: Compile parsers Unix like
         if: matrix.os != 'windows-2022'
         run: |
-          nvim --headless -c "TSInstallSync all" -c "q"
+          nvim --headless -c "TSUpdateSync all | TSInstallSync! c" -c "q"
 
       - name: Compile parsers Windows
         if: matrix.os == 'windows-2022'
         run: |
-          Neovim\\bin\\nvim.exe --headless -c "TSInstallSync all" -c "q"
+          Neovim\\bin\\nvim.exe --headless -c "TSUpdateSync all | TSInstallSync! c" -c "q"
```

Comment:
Equivalent to `:TSUpdate all` but probably more descriptive. We should probably try to install also the built-in parsers automatically so that they fit to our queries and we could for instance highlight C attributes without conflicting with different built-in parser versions of already released nvim-versions.